pkg/router/controller/contention.go

```diff
@@ -243,7 +246,10 @@ func (t *SimpleContentionTracker) Clear(id string, current *routev1.RouteIngress
 }
 
 func ingressEqual(a, b *routev1.RouteIngress) bool {
-	return a.Host == b.Host && a.RouterCanonicalHostname == b.RouterCanonicalHostname && a.WildcardPolicy == b.WildcardPolicy && a.RouterName == b.RouterName
+	// In addition to the RouteIngress' string fields, compare the available admission condition to determine
+	// if the given ingress' are equal. See https://bugzilla.redhat.com/show_bug.cgi?id=1908389.
```

Comment:
Whoops, this was presumably supposed to be https://bugzilla.redhat.com/show_bug.cgi?id=1903821.